x-pack/plugins/apm/public/components/shared/Links/DiscoverLinks/__test__/DiscoverLinks.integration.test.tsx

```diff
@@ -38,7 +38,7 @@ test('DiscoverTransactionLink should produce the correct URL', async () => {
   );
 
   expect(href).toEqual(
-    `/app/kibana#/discover?_g=(refreshInterval:(pause:true,value:'0'),time:(from:now%2Fw,to:now))&_a=(index:apm-index-pattern-id,interval:auto,query:(language:lucene,query:'processor.event:"transaction" AND transaction.id:"8b60bd32ecc6e150" AND trace.id:"8b60bd32ecc6e1506735a8b6cfcf175c"'))`
+    `/app/kibana#/discover?_a=(index:apm-index-pattern-id,interval:auto,query:(language:lucene,query:'processor.event:"transaction" AND transaction.id:"8b60bd32ecc6e150" AND trace.id:"8b60bd32ecc6e1506735a8b6cfcf175c"'))&_g=(refreshInterval:(pause:true,value:'0'),time:(from:now%2Fw,to:now))`
   );
```

Comment:
the _g and _a values just switched places based on how I'm now constructing the link, but they are otherwise the same all throughout this file